Makefile

```diff
@@ -601,3 +600,8 @@ diagrams: ## Build proposal diagrams
 .PHONY: serve-book
 serve-book: ## Build and serve the book with live-reloading enabled
 	$(MAKE) -C docs/book serve
+
+.PHONY: clean-envtest
+clean-envtest: ## Shutdown all envtest instances
+	-killall -9 etcd
+	-killall -9 kube-apiserver
```

Comment:
Any reason we need these killall directives? Shouldn't envtest shutdown at the end of each test?

Reply:
I think so. envtest should shutdown at the end of each test. In fact "make clean-envtest" does not find any process to kill ..